packages/solid/package.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "@web3-onboard/solid",
+  "version": "2.7.10",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
We will set the initial published staging package to 2.0.0 with an alpha flag to signify a prerelease for published testing of this package.